scripts/systemtap/kem/kemd.c

```diff
@@ -55,17 +54,16 @@ static void kemd_cleanup_module(void);
 static int kemd_open(struct inode *inode, struct file *filp);
 static int kemd_release(struct inode *inode, struct file *filp);
 static ssize_t kemd_read(struct file *filp, char *buf,
-			 size_t len, loff_t *offset);
+						 size_t len, loff_t *offset);
 static ssize_t kemd_write(struct file *filp, const char *buf,
-			  size_t len, loff_t *offset);
+						  size_t len, loff_t *offset);
 
 static struct file_operations kemd_fops = {
-	.owner   = THIS_MODULE,
-	.read    = kemd_read,
-	.write   = kemd_write,
-	.open    = kemd_open,
-	.release = kemd_release
-};
+	.owner = THIS_MODULE,
```

Comment:
The original code has a better alignment.

Reply:
aligned properly by putting the closing branch by it own line

Reply:
The extra spaces are *intentional* added to align all the '=' .